scripts/install.sh

```diff
@@ -1,15 +1,67 @@
+#!/bin/bash
 set -e
 
-if [ -f ./auto_node.sh ]; then
-    echo "Would you like to replace existing ./auto_node.sh (y/n)?"
+if [ "$(uname)" != "Linux" ]; then
+  echo "[AutoNode] not on a Linux machine, exiting."
+  exit
+fi
+if ! command -v systemctl > /dev/null; then
+  echo "[AutoNode] distro does not have systemd, exiting."
```

Comment:
why its an requirement, okay so what, I don't have systemd, or we are relying on it for the restart ability

Reply:
relying on it for restart ability. For now at least.